src/main/java/com/okta/tools/OktaAwsConfig.java

```diff
@@ -29,10 +33,11 @@ static OktaAwsCliEnvironment loadEnvironment() {
                 getEnvOrConfig(properties, "OKTA_ORG"),
                 getEnvOrConfig(properties, "OKTA_USERNAME"),
                 getEnvOrConfig(properties, "OKTA_PASSWORD"),
-                getEnvOrConfig(properties, "OKTA_PROFILE"),
+                getProfile(profile, getEnvOrConfig(properties, "OKTA_PROFILE")),
```

Comment:
👍 This is a neat solution. If we add more command-line switches in future, we can refactor, but this is the right solution for the current need.

Reply:
yes, tried to keep it simple